DepositeCalcTests/Pages/CalculatorPage.cs

```diff
@@ -64,7 +65,8 @@ public string FinancialYear
         public void Calculate()
         {
             CalculateBtn.Click();
-            new WebDriverWait(driver, TimeSpan.FromSeconds(2)).Until(_ => InterestEarnedFld.GetAttribute("value") != null);
+            new WebDriverWait(driver, TimeSpan.FromSeconds(2)).Until(_ => IncomeFld.GetAttribute("value") != "0,00" &&
+                                                                          InterestEarnedFld.GetAttribute("value") != "0,00");
```

Comment:
Instead of this wait you can wait until the calculate button is clickable again.